infra/staging/main.tf

```diff
@@ -39,6 +39,11 @@ variable "frontend_bucket_name" {
   default = "call-home-frontend"
 }
 
+variable "website_domain_name" {
+  type = string
+  default = "app2-staging.callhome.sg"
```

Comment:
Looks good :)

The name `website_domain_name` threw me off at first, but it's pretty accurate - this is the domain we want for the whole app, not just the frontend.